containers/package-builder/checkout-repositories-version.sh.j2

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+directory=$1
+repository=$2
+
+# Check if a directory exists
+if [ ! -d "$directory" ] && [[ "$repository" == "concent" ]]; then
+        git clone /var/concent-data/concent/ "$directory"
+        cd "$directory"
+        git checkout {{ concent_version }}
+fi
+
+if [ ! -d "$directory" ] && [[ "$repository" == "golem" ]]; then
+        git clone /var/concent-data/golem/ "$directory"
+        cd "$directory"
+        git checkout {{ golem_version }}
+fi
+
+if [[ "$repository" == "concent" ]]; then
+git fetch https://github.com/golemfactory/concent.git
+elif [[ "$repository" == "golem" ]]; then
+git fetch https://github.com/golemfactory/golem.git
```

Comment:
Don't fetch from github. You should fetch from the repository in `/var/`.

Reply:
Done.